soccer/planning/MotionInstant.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <Geometry2d/Point.hpp>
+
+namespace  Planning {
+/**
+ * This class represents a instant on a motion. It cointains a position and velocity vector
+ */
+    struct MotionInstant {
+
+        MotionInstant() : pos(), vel() {}
```

Comment:
Can we get rid of this constructor and just supply default values for the other constructor?